docs/semgrep-assistant/overview.md

```diff
@@ -49,42 +49,42 @@ By categorizing your code through component tags, Semgrep Assistant can help you
 
-You can view findings that Assistant suggests should be **Ignored** in Semgrep Cloud Platform's **Findings page**. <!-- #10438 -->
+![Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
+    *Figure*. Semgrep Assistant auto-triage in a Slack notification.
 
 ### Autofix Semgrep Code findings
 
-Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings identified using rules that don't have human-written autofix suggestions when it identifies a true positive. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, then Assistant will recommend its use in the autofix suggestion for the issue in your code.
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive, if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
 
-You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant.
+You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant in the **Settings** page in Semgrep Cloud Platform.
 
-This feature is only active in PR and MR comments. Developers can review and verify Semgrep's generated fixes before they're applied.
+Autofixes are available in Semgrep Cloud Platform's **Findings page** under **Your code** in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). They are also available in PR and MR comments, so developers can review and verify Semgrep's generated fixes before they're applied.
 
 ![Semgrep Assistant generating a potential fix](/img/semgrep-assistant-autofix.png)
 *Figure*. Semgrep Assistant generates a potential fix.
 
-You can view findings for which Assistant has generated autofix snippets in Semgrep Cloud Platform's **Findings page**. <!-- #10438 --> 
-
-![Sample finding entry with Semgrep Assistant verdict](/img/assistant-recommendations.png)
-
-The finding's detail page also includes a link to the PR or MR comment with the autofix; you can go to the PR or MR, where you can directly commit the autofix. <!-- #10516 --> <!-- This isn't unique to Assistant; it's available for autofix and PR/MR comments in general. -->
+The finding's details also include a link to the PR or MR comment with the autofix, so you can go to the PR or MR, where you can directly commit the autofix.
```

Comment:
I'd like to find a screenshot for this, but I haven't been able to see it happening currently.

Reply:
Did you mean a more specific link than something in the Activity section? 

![image](https://github.com/semgrep/semgrep-docs/assets/16521651/0d5e4c70-167d-41d0-a858-4c27e52d9e89)

Reply:
It's possible I misunderstood what this is referring to; I was thinking it was part of the info specific to the autofix (which seems not working, as mentioned in my later comment). But if it's only referring to a link to the PR (not the comment specifically - that link will just take you to the PR in general) then this would work.

Reply:
@jsmoxon can you please clarify what the expected behavior for this is?

Reply:
The expected behavior is that where possible, we show an AI autofix code snippet on the Findings details page. We also offer a "copy" snippet button so that users can share it with developers. However, I think this bug is preventing any autofix snippets from being shown: https://linear.app/semgrep/issue/AI-505/autofix-not-showing-in-finding-details-for-findings-even-if-they-have 

@underyx do we include a link to the PR/MR as well?

Reply:
As Katie brought up, we have a link to the PR/MR in the Activity - the question is more if there ought to be a link that's specifically for going to the comment containing the autofix (this is what I thought it was saying). Or if the wording is ambiguous and it's actually referring to the main link to the PR (which is likely fine, but would benefit from clearer wording so other folks don't get on the track I did).

Reply:
I changed the wording here a bit so it's clear we link to the PR/MR where the comment exists.